### PR TITLE
Webpack conf cleanup

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: config/dev.webpack.config.js [hunk 1]
```diff
@@ -4,15 +4,14 @@ const { GitRevisionPlugin } = require('git-revision-webpack-plugin');
...
-    deployment: process.env.BETA ? 'beta/apps' : 'apps',
     useProxy: true,
     proxyVerbose: true,
-    debug: true,
     env: `${process.env.STAGE ? 'stage' : 'prod'}-${process.env.BETA ? 'beta' : 'stable'}`,
-    appUrl: process.env.BETA ? '/beta/insights/image-builder' : '/insights/image-builder',
 };
 
 const { config: webpackConfig, plugins } = config({
+    deployment: process.env.BETA ? 'beta/apps' : 'apps',
```
Comment (jkozol) [inline config/dev.webpack.config.js:13]: Should the prod config also default to apps? Currently it is `beta/apps` or not default. Is default apps?

File: config/dev.webpack.config.js [hunk 2]
```diff
@@ -4,15 +4,14 @@ const { GitRevisionPlugin } = require('git-revision-webpack-plugin');
...
+    appUrl: process.env.BETA ? '/beta/insights/image-builder' : '/insights/image-builder',
```
Comment (jkozol) [inline config/dev.webpack.config.js:14]: Also, why do we not need to declare `appUrl` in the prod config?

**Review by jkozol — APPROVED** [review]
Overall makes sense. Just a couple questions

File: config/dev.webpack.config.js (hunk 1 above)
Comment (Gundersanne) [reply config/dev.webpack.config.js:13]: Yea when we deploy to prod-stable it should also default to `apps`. I'll add this, one less thing to think about when we get there.

File: config/dev.webpack.config.js (hunk 2 above)
Comment (Gundersanne) [reply config/dev.webpack.config.js:14]: Ugh this is a mistake sorry, appURL is proxy sepcific; deployment isn't. Changing this.

**Review by jkozol — APPROVED** [review]

**Pull request merged**